packages/shared/lib/core/wallet/utils/getAccoutByAddress.ts

```diff
@@ -0,0 +1,11 @@
+import { findAccountWithAddress } from '@lib/wallet'
+import { Recipient, Sender } from '../types'
+
+export function getAccoutByAddress(address: string): Recipient | Sender {
+    const account = findAccountWithAddress(address)
+    if (account) {
+        return { type: 'account', account: account }
+    } else {
+        return { type: 'address', address: address }
+    }
+}
```

Comment:
call this getSubjectFromAddress and then we can type Subject as Recipient | Sender, if we don't already have this subject type. or even replace the Recipient and the Sender type with one Subject type?